Revit_Core_Engine/Convert/Structure/FromRevit/Bars.cs

```diff
@@ -103,12 +102,20 @@ public static List<Bar> BarsFromRevit(this FamilyInstance familyInstance, RevitS
                 if (profile.Edges.Count == 0)
                     familyInstance.Symbol.ConvertProfileFailedWarning();
 
+                if (!materialFound)
+                    Compute.InvalidDataMaterialWarning(familyInstance);
+
                 property = BH.Engine.Structure.Create.SectionPropertyFromProfile(profile, materialFragment, profileName);
             }
             else
             {
                 property = property.GetShallowClone() as ISectionProperty;
-                property.Material = materialFragment;
+
+                if (!materialFound)
+                    BH.Engine.Reflection.Compute.RecordWarning($"Section property of a bar has been loaded from the library, but the material could not be converted. Default material of the section property has been used. Revit ElementId: {familyInstance.Id.IntegerValue}");
```

Comment:
this sounds like the library material could not be converted, which to the average user sounds like a big problem. In fact, the likely case is that the user did not specify a material in Revit, which might not be a problem at all.